NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -90,6 +91,17 @@ class NeuroID
         internal var locationService: LocationService? = null
 
         init {
+            when (serverEnvironment) {
+                "production" -> {
+                    endpoint = "${Constants.productionEndpoint.displayName}"
+                    scriptEndpoint = "${Constants.productionScriptsEndpoint.displayName}"
+                }
+                "development" -> {
+                    endpoint = "${Constants.devEndpoint.displayName}"
+                    scriptEndpoint = "${Constants.devScriptsEndpoint.displayName}"
+                }
+            }
```

Comment:
if a customer sends in an invalid value will that cause the endpoints to be wrong/not have a value? or will the defaults be to production regardless of a value being passed in that is valid or not?

Reply:
good point. will try.

Reply:
it will drop out of the when and use production defaults. that is the reason why when we had "prod" in the sample app, it didn't match any of the clauses, drop out of the when block. since production is set by default, it used default. we will be good.